lukasz-lapczyk/review-dealer
# Add company reviews

# Overview
Added Review model (containing body - 5-500 characters and rating - 0-5 value). Registered users can add reviews to companies - one review to each company, except their own. 

# Testing prerequisities
- bundle install
- db:migrate

# Testing 
- navigate to root directory
- log in with 'user_1@example.org', pass: Secret19 or create new account
- create new company
- log out and log in as 'user_2@example.org', pass: Secret29 or create new account
- click on previously created company's name
- click 'Add review' button
- fill in form with valid data and click Create
- [x] review should appear on list
